.github/maintainers_guide.md

```diff
@@ -150,26 +149,34 @@ password: {your password}
 
 3. (Slack Internal) Communicate the release internally
 
-
 #### Production Deployment
 
 1. Create the commit for the release:
    - Bump the version number in adherence to [Semantic Versioning](http://semver.org/) in `slack_bolt/version.py`
    - Build the docs with `./scripts/generate_api_docs.sh`.
    - Commit with a message including the new version number. For example `1.2.3` & Push the commit to a branch and create a PR to sanity check.
-     - `git checkout -b v1.2.3-release`
-     - `git add --all`
-     - `git commit -m 'version 1.2.3'`
-     - `git push {your-fork} v1.2.3-release`
-   - Merge in release PR after getting an approval from at least one maintainer.
-   - Create a git tag for the release. For example `git tag v1.2.3`.
-   - Push the tag up to github with `git push origin --tags`
+     - `git checkout -b v1.2.3`
+     - `git commit -a -m 'version 1.2.3'`
+   - Open a PR and merge after receiving at least one approval from other maintainers.
 
 2. Distribute the release
    - Use the latest stable Python runtime
```

Comment:
```suggestion
   - Use the latest stable Python runtime
      - `python --version`
```

QQ: Do the scripts below make sure the latest `python` is used? That's a little unclear to me, but no blocker!

Reply:
No they do not, in some situations we may want to build with versions that are not the latest

But I like this recommendation 💯